client/components/ReportsPanel/ReportsTabbar.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component, PropTypes } from 'react';
+
+import './style.scss';
+
+const Link = props => (
```

Comment:
Yes, like I said above: much of this will change in the next sprint, when we introduce routes.